app/views/layouts/_google_analytics.html.erb

```diff
@@ -1,6 +1,13 @@
 <!-- Global site tag (gtag.js) - Google Analytics -->
 <script async src="https://www.googletagmanager.com/gtag/js?id=<%= SITE_CONFIG['google_analytics_script_id'] %>"></script>
 
+  <script>
+    dataLayer = [{
+      'signed_in': '<%= user_signed_in?.to_json %>'
+      'hasIPs': '<%= (!current_user.nil? && current_organisation.ips.any?).to_json %>'
```

Comment:
You won't need either of the `.to_json` as `false.to_json` => 'false' and  '<%= false %>' => 'false'.

also `!current_user.nil?` can just be written as `!!current_user` as `!!nil` => `false`

Would also use `signedIn` javascript style rather than `signed_in` ruby style considering its js.

Reply:
Added `.to_json` because of the ERB linter.

Reply:
also needs a comma in between the two hash key value pairs.